example/render_window_display_async.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
```

Comment:
I didn't see any window displaying. Does this behavior right for this test file?

Reply:
This test last only few seconds, maybe you did not see it.
The test isn't displaying anything in the window, it's just showing req in the console to see if that's actually updating properly ^^